src/Application/index.ts

```diff
@@ -0,0 +1,6 @@
+export * from './ApplicationCreation';
+export * from './ApplicationEdition';
+export * from './ApplicationManagement';
```

Comment:
I don't think we need to barrel these components. Realistically, `ApplicationManagement` should be the only one available to the other components, with all the other ones only serving as internal helpers. Although an alternative I could think of is just renaming the main component from `ApplicationManagement` to `index.tsx` for easier exports?

Reply:
Removed index files